.gitpod.yml

```diff
@@ -18,7 +18,6 @@ tasks:
 
         yarn install
 
-        rails g webpacker:install && rails g webpacker:install:react && rails g react:install
```

Comment:
I think i would guess it's `2) webpack has not yet re-run to reflect updates` -- so, if we remove the `g` lines, which of these three ought we to keep if any?

`rails webpacker:install && rails webpacker:install:react && rails g react:install`